.github/workflows/bicep_workflow_feature.yml

```
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - feat/stbr-infra
+      - feat/FN-974/create-environment-variable-pdc-inputter-email
     paths:
       - '.github/workflows/bicep_workflow.yml'
       - '.github/workflows/bicep_deploy.yml'
```

should these be this instead?
```suggestion
      - '.github/workflows/bicep_workflow_feature.yml'
      - '.github/workflows/bicep_deploy.yml'
      - '.github/workflows/bicep_az_deployment_group.yml'
```